src/Raven.Client/Exceptions/ExceptionDispatcher.cs

```diff
@@ -24,6 +25,8 @@ internal sealed class ExceptionSchema
 
             public string Type { get; set; }
 
+            public LimitType? LicenseLimitSubType { get; set; }
```

Comment:
no need to introduce this, check how this is handled in 'MaybeAddAdditionalExceptionData'

Reply:
Done